CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.0 
+
+* Change fields ResourceGuaranteedCpu, VCpuInMhz, IsThinProvision, NetworkPoolReference,ProviderVdcReference and UsesFastProvisioning in AdminVdc to pointers
```

Comment:
Please add also why this change is needed. Also, there's a space missing after one comma.

Reply:
done